indykite/common_schema.go

```diff
@@ -175,3 +176,11 @@ func deletionProtectionSchema() *schema.Schema {
 		Description: `Whether or not to allow Terraform to destroy the instance. Unless this field is set to false in Terraform state, a terraform destroy or terraform apply that would delete the instance will fail.`,
 	}
 }
+
+func regionSchema() *schema.Schema {
+	return &schema.Schema{
+		Type:        schema.TypeString,
+		Required:    true,
+		Description: `Region where the application space is located.`,
```

Comment:
How region actually work? Will it be possible to specify each entity region? Like Application, AppAgent etc?

If not, then move it to AppSpace. Otherwise change Description, because it seems that it is AppSpace related.

Reply:
can be set only for app_space